modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/util/SOAPSecurityHandler.java

```diff
@@ -27,15 +28,19 @@
 import java.util.HashSet;
 import java.util.Set;
 
+/**
+ * Class for handle SOAP message security.
```

Comment:
As per prabath's comments, this could be "Class to handle SOAP message security."